src/main/java/bio/terra/pdao/PrimaryDataAccess.java

```diff
@@ -29,7 +29,7 @@
  *     </ol>
  * </ul>
  *
- * All implementation-specific exceptions should be caught by the implementation and wrapped
+ * All implementation-specific exception should be caught by the implementation and wrapped
```

Comment:
seems like an unnecessary change

Reply:
Yes, I did a rename and it went too far. I caught a few of these. I'll fix this one, also. Thanks